polars/tests/it/io/csv.rs

```diff
@@ -340,6 +340,17 @@ fn test_new_line_escape() {
     let _df = CsvReader::new(file).has_header(true).finish().unwrap();
 }
 
+#[test]
+fn test_new_line_escape_on_header() {
+    let s = r#""length","header with
+ new line character", "width"
+ 5.1,3.5,1.4
+ "#;
+    println!("{}", "ok");
+    let file: Cursor<&str> = Cursor::new(s);
+    let _df: DataFrame = CsvReader::new(file).has_header(true).finish().unwrap();
```

Comment:
I think we should assert the output of the `DataFrame` to be correct for this test to have utility.

Reply:
I modified an assertion to the test. Review when you have time, please :)